src/main/java/spoon/reflect/meta/impl/ModelRoleHandlers.java

```diff
@@ -1,4 +1,4 @@
-/*
+/**
```

Comment:
Why is this a javadoc comment now? The other license headers seem to be normal comments.

Reply:
as you see in the CI the header is wrong, I just need to apply the license plugin

Reply:
Alright :) Looked a bit odd at first glance...